identity/src/main/java/com/android/identity/Constants.java

```diff
@@ -111,6 +111,11 @@ public class Constants {
      * for BLE data retrieval.
      */
     public static final int BLE_DATA_RETRIEVAL_OPTION_MDOC_PERIPHERAL_SERVER_MODE = (1 << 1);
+    /**
+     * Flag indicating that the <em>mdoc BLE L2CAP</em> should be used if supported
+     * for BLE data retrieval.
+     */
```

Comment:
I think saying "Flag indicating that L2CAP should be used for data retrieval if available and supported." is a bit more precise.